.buildkite/pipeline.yml

```
@@ -1,5 +1,65 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 
 steps:
-  - label: "Example test"
-    command: echo "Hello!"
+
+  - label: ":linux: Tooling test"
+    env:
+      SETUP_GOLANG_VERSION: "{{matrix.go_version}}"
+    command:
+      - ".buildkite/scripts/tooling_test.sh"
```

missing the matrix field:
```suggestion
      - ".buildkite/scripts/tooling_test.sh"
    matrix:
      setup:
        go_version:
          - "1.15.12"
          - "1.16.4"
```